libmscore/instrtemplate.cpp

```diff
@@ -312,6 +329,8 @@ void InstrumentTemplate::write(XmlWriter& xml) const
             if (!isGlobal)
                   ma.write(xml);
             }
+      if (!family)
+            xml.tag("family", family->id);
```

Comment:
I think this is a bug, must be 
```
if (family)
            xml.tag("family", family->id);
```

Reply:
Outch!

Reply:
Oops! Luckily it is a problem when you manage to access this via an invisible form on the Instrument Dialog :wink: 
Nevertheless, it is a bug. And solved in PR #7405 .